psalm.xml.dist

```diff
@@ -43,6 +43,17 @@
                 <referencedClass name="GdImage" />
             </errorLevel>
         </UndefinedClass>
+
+        <UndefinedAttributeClass>
```

Comment:
I would say that we add this to the baseline: this is not something we explicitly ignore, but rather will disappear with the PHP version upgrade